### DISCUSSION TITLE
How to contribute mappings?

### DISCUSSION THREAD

**Discussion by Fincap** [MCPHackers/RetroMCP-Java#126] [Q&A]
If I have deduced the meaning of obfuscated symbols that aren't included by the deobfuscator, what is the most effective way to contribute my findings to this application (or whichever dependency contains the mappings)?

**Comment by PhoenixVX** [comment]
Please refer to the other issues detailing how this is done.

**Reply by Fincap** [reply]
Which other issues? The only one I can find that is sort-of related is #21. Should I create a pull request into https://github.com/MCPHackers/MCPHackers.github.io/tree/main/versionsV2 with an updated zip? Are these zips generated from another repository/project?
